.github/workflows/mirror.yml

```diff
@@ -0,0 +1,33 @@
+name: Mirror Image
```

Comment:
do we need a separate action for this?

Reply:
this is helpful for doing a one time trigger for older images, for eg. the current version used by cli `v0.21.4`. Using this job, I can mirror it over from docker hub to ghcr without rebuilding.